lib/task-data/base-tasks/dell-wsman-add-hotspare-updatexml.js

```diff
@@ -0,0 +1,12 @@
+// Copyright © 2017 Dell Inc. or its subsidiaries. All Rights Reserved.
+
+'use strict';
+
+module.exports = {
+    friendlyName: "Dell Wsman Add Hotspare UpdateXml Base Task",
+    injectableName: "Task.Base.Dell.Wsman.Add.Hotspare.UpdateXml",
```

Comment:
Just curious, if this UpdateXml is only used for AddHostSpare, you should not create a job for it but to put in into add host spare job.

Reply:
It just follow the code structure design as how add-volume and delete-volume did, and reuse the common code.

Reply:
Basically, A task for RackHD should be able to be executed independently with input options, it should not be coupled with other tasks.

Reply:
Agree. But I prefer to create another story, to refactor these code structure in add hot spare, add volume and delete volume features.
@yaolingling